.github/CONTRIBUTING.rst

```diff
@@ -31,6 +31,9 @@ also suggestions on how you can most effectively help the project.
 Please be aware that our workflow does deviate slightly from the typical GitHub
 project. Details on how to properly submit a pull request are covered in
 `Lifecycle of a Pull Request <https://cpython-devguide.readthedocs.io/pullrequest.html>`_.
+One key point is to keep comments on GitHub to only those related to the reviewing
+the code in the pull request. All other discussions -- e.g. about the issue being
```

Comment:
So I thought about that, and the problem is "other discussions" is "everything but code review". 😉 So I just couldn't think of a better phrasing. Obviously feel free to tweak it if you have any ideas.